src/confopt/oneshot/archsampler/composite_sampler.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+import torch
+
+from confopt.oneshot.archsampler import BaseSampler
+from confopt.oneshot.base import OneShotComponent
+
+
+class CompositeSampler(OneShotComponent):
+    def __init__(
+        self,
+        arch_samplers: list[BaseSampler],
+        arch_parameters: list[torch.Tensor],
+    ) -> None:
+        super().__init__()
+        self.arch_samplers = arch_samplers
+        self.arch_parameters = arch_parameters
+
+        # get sample frequency from the samplers
+        self.sample_frequency = arch_samplers[0].sample_frequency
```

Comment:
Do we still use sampling frequency in the code?

Reply:
yes, we never change it to epoch in our experiments, but we have option to do that. I'm not sure how handy this can come later, but I think it doesn't hurt to leave it there